rfcs/0077-stale-issues-amendment.md

```diff
@@ -0,0 +1,135 @@
+---
+feature: stale-issues-amendment
+start-date: 2020-10-13
+author: blaggacao
+co-authors: (to be found)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/nixpkgs/pull/100460, https://github.com/NixOS/nixpkgs/pull/100462
+---
+
+# Summary
+[summary]: #summary
+
+RFC 51 implemented the stale bot with the following motivation:
+
+> We have a large number of open issues that have accumulated over the years. Not all of them are still valid and need our attention.
+> 
+> By marking stale issues, we can more easily filter issues for ones that have at least one person interested in them.
+
+Under this motivation, the inactivity period was chosen to be 180 days.
+
+This RFC proposes an alternative interpretation of "stale":
+
+_A stale issue (or PR) is an issue on which the discussion has went stale._
+
+_Explicitly, it has no other semantic than this. It is a neutral 
+aggregated indicator of facts. It reflects the fact that 
+individuals did choose not to interact on a particular issue or PR
+for an extended period of time._
+
+_180 days is too long of such extended period of time. People do relate
+"stale" with some shorter time frame. Therefore, a PR goes stale after
+a 90 days period (vs previously 180 days) and an issue goes stale after
+a 60 days period (vs previously 180 days)._
+
+# Motivation
+[motivation]: #motivation
+
+Under the renewed definition of _stale_, people are betrayed. 
+
+They are not told the truth, though the facts are long evident.
+
+A majority of issues or PRs with no interaction for quite short period
+of times already (maybe 3-6 weeks) has a lesser change of "success".
+
+People with vested interest should see the _stale_ label and be prompted
+to think: "Oh, I need this, too. Damn, it's stale. Let's have a look
+and do something to help out."
+
+An issue that hasn't been interacted with for 60 days or a PR for 90 days
+and in the vast majority of cases deserves this hint out of fariness.
+
+Honesty and transparency are by far more effective than any perceived
+political correctness (judging by the discussions on RFC 51).
```

Comment:
This sentence seems more like a subjective complaint rather than an actual reasoning why you disagree with the results of the mentioned RFC.

Reply:
It is a claim anyone can disagree. I hope dearly hope though that honesty is higher rated in the nix community than political correctness. Would you disagree?

Reply:
Yes, everyone can disagree if they want to. However, this is a technical document about a change affecting our ecosystem, so this is IMHO not the place to write down personal impressions as if those would be a widely-accepted truth.

Reply:
I agree, but it is the motivation section. What's wrong about being motivated based on a personal judgment? But I agree it should not do this:
> write down personal impressions as if those would be a widely-accepted truth.

Reply:
I reworded in https://github.com/NixOS/rfcs/pull/77/commits/120af202277c13d6bb084adbb614464807496b8b

Hence marking this particular feedback as resolved. Thanks, the document is better now! Feel free to reopen, if there is anything more to it...